UI.Web/ClientApp/src/app/components/tic-tac-toe/board/board.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'board',
+    templateUrl: './board.component.html',
+    styles: [
+        `main {display: grid;
+    grid-template-columns: 200px 200px 200px;}`
+    ]
+})
+
+export class BoardComponent implements OnInit {
+    squares: any[];
+    xIsNext: boolean;
+    winner: string;
+
+    constructor() {}
+
+    ngOnInit() {
+        this.newGame();
+    }
+
+    newGame() {
+        this.squares = Array(9).fill(null);
```

Comment:
Also very interesting and simplified from what we did previously.  Instead of coding the game in the way we see it visually like we did with a 2D array of 3 rows of 3 columns this is just a normal array of 9 spots and then it's modified to visually look like it's 3x3